src/App.js

```diff
@@ -13,24 +11,38 @@ import NewProject from './pages/NewProject';
 import NewClient from './pages/NewClient';
 import Tasks from './pages/Tasks';
 import Users from './pages/Users';
+import Header from './components/Header';
 
 function App() {
+  const LoginContainer = () => (
```

Comment:
thank you for dividing container so we don't have to put <Header /> everywhere

Reply:
^_^